plugins/examples/custom-processor/src/main/java/org/elasticsearch/example/customprocessor/ExampleRepeatProcessor.java

```diff
@@ -44,7 +45,8 @@ public ExampleRepeatProcessor create(
             Map<String, Processor.Factory> registry,
             String tag,
             String description,
-            Map<String, Object> config
+            Map<String, Object> config,
+            ProjectId projectId
```

Comment:
Not related to custom authz engine. But needs to be fixed as well so that CI can pass.